src/components/BulkEditPane/BulkEditMarcLayer/BulkEditMarcLayer.js

```diff
@@ -40,11 +47,20 @@ export const BulkEditMarcLayer = ({
   const sortedOptions = sortAlphabetically(options);
 
   const marcFormErrors = getMarcFormErrors(marcFields);
+  const marcContentUpdates = marcFields.map(getTransformedField);
+  const marcContentUpdatesWithoutId = marcContentUpdates.map(item => omit(item, ['id']));
   const contentUpdates = getMappedContentUpdates(fields, options);
 
-  const isMarcFieldsValid = Object.keys(marcFormErrors).length === 0;
+  const isMarcFormValid = isMarcContentUpdatesFormValid(marcFormErrors);
   const isAdministrativeFormValid = isContentUpdatesFormValid(contentUpdates);
-  const isAnyFormValid = isMarcFieldsValid || isAdministrativeFormValid;
+
+  const isAdministrativeFormPristine = !isEqual(ADMINISTRATIVE_FORM_INITIAL_STATE, contentUpdates);
+  const isMarcFormPristine = !isEqual(MARC_FORM_INITIAL_STATE, marcContentUpdatesWithoutId);
+
+  // state is valid if at least one form is filled and valid and another one is not pristine or both forms are filled and valid
```

Comment:
I think you meant to say in the comment that `another one is pristine`

Reply:
I'm with @BogdanDenis here, but I feel like both the comment AND the conditional contradict the PR description which states "2 valid forms, or 1 valid and one untouched". Here, you instead check 2 valid forms or 1 valid and one NOT pristine. Or do you have the booleans on lines 57 and 58 configured backwards, i.e. `isPristine` should be true when those values are _equal_, instead of when they differ? 

Super petty: since your comment describes two different scenarios, just put them on two different lines. As-is, it's hard to parse how the ands and ors separate things.